addons/l10n_in_edi/models/account_move.py

```diff
@@ -242,6 +270,22 @@ def _l10n_in_edi_send_invoice(self):
             'company_id': self.company_id.id,
         })
         self.l10n_in_edi_status = 'sent'
+        message = []
+        if partner_validation := self._l10n_in_validate_einvoice_optional_field(self.partner_id):
+            message.append(Markup("").join(partner_validation))
+        if company_validation := self._l10n_in_validate_einvoice_optional_field(self.company_id.partner_id):
+            message.append(Markup("").join(company_validation))
+        if message:
+            message = Markup(
+                "<div class='alert alert-warning'>"
+                "<strong>{}</strong><br/>"
+                "{}"
+                "</div>"
+            ).format(
+                _("Optional Field Validation Warnings:"),
+                Markup("").join(message)
```

Comment:
```suggestion
                "".join(message)
```
Not required, it will already go to the Markup